examples/multiple_clusters.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kubernetes import client, config
+# install pick using "pip install pick". It is not included
+# as a dependency because it only used in examples
+from pick import pick
+
+
+def main():
+
+    contexts, active_context = config.list_kube_config_contexts()
+    if not contexts:
+        print("Cannot find any context in kube-config file.")
+        return
+    contexts = [context['name'] for context in contexts]
+    active_index = contexts.index(active_context['name'])
+    cluster1, first_index = pick(contexts, title="Pick the first context",
```

Comment:
Yes, it lets user pick two clusters and then show list of pods in them. I am going to extend this example in future, maybe letting user copy a resource from one cluster to the other.